include/zephyr/drivers/uart.h

```diff
@@ -124,6 +124,10 @@ struct uart_config {
 	uint8_t stop_bits;  /**< Stop bits, use @ref uart_config_stop_bits */
 	uint8_t data_bits;  /**< Data bits, use @ref uart_config_data_bits */
 	uint8_t flow_ctrl;  /**< Flow control setting, use @ref uart_config_flow_control */
+#ifdef CONFIG_SERIAL_SUPPORT_DATA_LINE_INVERSION
+	bool rx_invert;		/**< enable rx logic inversion */
+	bool tx_invert;		/**< enable tx logic pin inversion */
+#endif
```

Comment:
Is run-time configuration of rx/tx inversions really needed? Wouldn't this always be known at compile-time (due to hardware design), allowing us to keep these two booleans internally in the driver - and not needing the Kconfig?